deliver/lib/deliver/download_screenshots.rb

```diff
@@ -13,7 +13,7 @@ def self.run(options, path)
     end
 
     def self.download(options, folder_path)
-      v = options[:use_live_version] ? options[:app].live_version : options[:app].latest_version
+      v = options[:use_live_version] ? options[:app].live_version(platform: options[:platform]) : options[:app].latest_version(platform: options[:platform])
```

Comment:
Could you explain to need this change?

Reply:
`platform` could be `nil`. https://github.com/fastlane/fastlane/blob/7da852267f38ab7e08ccc58b6cbc31d39dc6089f/deliver/lib/deliver/options.rb#L81
Do you need to care optional?

Reply:
this was related to this issue https://github.com/fastlane/fastlane/issues/14547 .
when I am giving value for platform as appletvos it is not getting passed. So options[:app].latest_version(platform: options[:platform]) output "v" for platform "ios".

Will it break something else?

Reply:
I have tested it. it works fine.

Reply:
@giginet meant that `options[:platform]` could be nil, thus giving `platform: nil` to `live_version` which might be a problem?

Reply:
Default argument in live_version is platform:nil. If options [:platform] is nil then that will get pass. which was happening anyways before. Problem was when options [:platform] is not nil and it did not get pass in live_version method call.